src/main/java/ru/turlyunef/core/FileReaderWriter.java

```diff
@@ -7,8 +7,6 @@
 import java.io.*;
 import java.util.ArrayList;
 
-import static java.lang.Integer.parseInt;
-
 public class FileReaderWriter {
```

Comment:
It is Bad name for this class - maybe better converter, or create two classes like Reader and Writer. But you don't have there only methods - read and write. Think about it